sktime/forecasting/base/adapters/_fbprophet.py

```diff
@@ -10,15 +10,20 @@
 import pandas as pd
 
 from sktime.forecasting.base._base import DEFAULT_ALPHA
-from sktime.forecasting.base._sktime import _OptionalForecastingHorizonMixin
-from sktime.forecasting.base._sktime import _SktimeForecaster
+from sktime.forecasting.base import BaseForecaster
 from contextlib import contextmanager
 
 
-class _ProphetAdapter(_OptionalForecastingHorizonMixin, _SktimeForecaster):
+class _ProphetAdapter(BaseForecaster):
     """Base class for interfacing fbprophet and neuralprophet"""
 
-    def fit(self, y, X=None, fh=None, **fit_params):
+    _tags = {
+        "univariate-only": True,
```

Comment:
why is this univariate only? `fbprophet` is can deal with exog data? also `neuralprophet` which might inherit from this class if s.o. writes a wrapper.

Reply:
Thank you @aiwalter. I would edit it soon

Reply:
yes, good point!

Can prophet also do multivariate?

If it can, I would say: let's restrict ourselves to refactoring here, and let's make another issue that addresses the multivariate case after the multivariate extension.

Reply:
afaik it can do only use `X`, but not do vector forecast. S.o. would have to check this. If it can forecast also `Y`, this would be definitely a separate PR, but changing the tag for now to accept `X` should be possible I guess

Reply:
I thought the "univariate" tag was about `y`, not `X`

Reply:
@aiwalter  does s.o. mean so?

Reply:
hm, I think `univariate-only` is about if `X` is used or not. Where are those tags described?

Reply:
s.o. = someone

Reply:
So, the conclusion is to leave it as `True`

Reply:
you can make it `True` just now, as we know this is the case